services/s3/src/main/resources/codegen-resources/customization.config

```diff
@@ -51,5 +51,12 @@
     "GetBucketNotification",
     "PutBucketLifecycle",
     "PutBucketNotification"
-  ]
+  ],
+  "utilitiesMethod": {
+    "name": "utilities",
```

Comment:
Should we remove this param so there's no chance of them becoming inconsistent across services? Will we ever not name these `utilities`?

Reply:
Added incase there is a need to use a new name. I can remove it for now though to ensure consistency.

Reply:
+1